src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -22,22 +22,17 @@ describe("Shipping", () => {
     )
   }
 
-  let props: ShippingProps
+  let testProps: ShippingProps
```

Comment:
@mzikherman, @damassi - Lint would not let me commit because of the shadowed props error. Was wondering if it was supposed to be repressed locally but ended up renaming vars in test since I think it actually improves test readability. Let me know if you think otherwise.

Reply:
Shadowed vars oftentimes use an `_` but I'm not particular about this in tests.

Reply:
Happy to update that as well:)

Reply:
Oh no that's ok! This is mostly applies to one-off stuff, but i see you updated in a few places. Its all good 😄